sdk/python/kfp/dsl/_ops_group.py

```diff
@@ -92,10 +126,26 @@ class Condition(OpsGroup):
   def __init__(self, condition):
     """Create a new instance of ExitHandler.
     Args:
-      exit_op: an operator invoked at exiting a group of ops.
+      condition (ConditionOperator): the condition.
 
     Raises:
       ValueError is the exit_op is invalid.
     """
     super(Condition, self).__init__('condition')
     self.condition = condition
+
+class Graph(OpsGroup):
```

Comment:
This should probably be eventually replaced with [ComponentSpec](https://github.com/kubeflow/pipelines/blob/master/sdk/python/kfp/components/_structures.py#L228) with [GraphSpec](https://github.com/kubeflow/pipelines/blob/master/sdk/python/kfp/components/_structures.py#L488) implementation.

Reply:
We have not reviewed the GraphSpec yet. We can plan and replace it once we have the roadmap.
Thanks